azure-storage-queue/src/main/scala/akka/stream/alpakka/azure/storagequeue/scaladsl/AzureQueueSink.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.azure.storagequeue.scaladsl
+
+import com.microsoft.azure.storage.queue.{CloudQueue, CloudQueueMessage}
+import akka.stream.alpakka.azure.storagequeue.{AzureQueueSinkFunctions, DeleteOrUpdateMessage, FlowMapECStage}
+import akka.stream.scaladsl.{Flow, Keep, Sink}
+import akka.Done
+import scala.concurrent.{ExecutionContext, Future}
+
+object AzureQueueSink {
+
+  /**
+   * ScalaAPI: creates a [[akka.stream.scaladsl.Sink]] which queues message to an Azure Storage Queue.
+   */
+  def apply(cloudQueue: CloudQueue, maxInFlight: Int = 4): Sink[CloudQueueMessage, Future[Done]] =
```

Comment:
Magical number here, why 4 is a good default? Should it be a constant somewhere - (with an explanation attached)?